onboarding/general-member.md

```diff
@@ -0,0 +1,68 @@
+# General Member onboarding
+
+Welcome to the TODO Group!
+
+This guide should provide your company representatives helpful onboarding
+resources once your [TODO General Membership Application](https://joinnow.todogroup.org/)
+has been validated and processed.
+
+An issue should have been created to process your company's membership (via this [template](/.github/ISSUE_TEMPLATE/onboarding-general-member.md)).
+
+If you run into any issues during the onboarding process, please reach out to the TODO Group Program Manager ([@anajsana][todo-pm]).
+
+- [Communication channels](#communication-channels)
+  - [Registration required](#registration-required)
+  - [Social media](#social-media)
+  - [GitHub](#github)
+  - [Private channels](#private-channels)
+- [Meetings and Calls](#meetings-and-calls)
+  - [Public calls](#public-calls)
+  - [Private calls](#private-calls)
+- [Resources and initiatives](#resources-and-initiatives)
+
+## Communication channels
+
+<!--
+TODO: Publish this info on the website to de-duplicate content
+-->
+
+### Registration required
+
+- [TODO Group Slack](https://slack.todogroup.org/)
+- [Community Mailing List](https://docs.google.com/forms/d/e/1FAIpQLSeU0YGM_IJ6gY8E5IIiwXKD_FZi3kAVc4E9_-3dtTDyKMSjdA/viewform)
+- [OSPO Newsletter](https://www.getrevue.co/profile/osponews)
+- [OSPOlogy](https://github.com/todogroup/ospology/discussions) (OSPO Forum Discussions)
+
+### Social media
+
+- [Twitter](https://twitter.com/todogroup)
+- [LinkedIn](https://www.linkedin.com/company/todo-group/)
+
+### GitHub
+
+- Verify membership to [TODO Group GitHub Organization](https://github.com/orgs/todogroup/people)
+
+### Private channels
+
+- [General Member mailing list](https://groups.google.com/g/private-todogroup)
+- General Member Slack channel: [#private](https://thetodogroup.slack.com/archives/G5TBGTX29)
+
+## Meetings and Calls
+
+### Public calls
+
+- [OSPOlogy Community Calls](https://community.linuxfoundation.org/todo-group/) (Monthly on Wednesdays)
+- TODO [EU Sync Meetings](https://community.linuxfoundation.org/todo-group-europe/) (Monthly on Thursdays)
+
+### Private calls
+
+- General Members Touchpoint (an invite is sent once you are added to the TODO private mailing list): Bi-weekly informal meetings on Fridays
+- General Members AMA session (an invite is sent once you are added to the TODO private mailing list): Quarterly on mondays
+
+## Resources and initiatives
+
+- Review the existing GitHub repos in the TODO Group org's [`README.md`](https://github.com/todogroup)
+- Read TODO Group [FAQ](https://todogroup.org/faq/)
+- Read the [TODO Charter](https://github.com/todogroup/governance/blob/main/CHARTER.adoc)
+
```

Comment:
I would like to add an extra section called "First-time contributions" to at point new members to contribution guidelines and repos where they can start adding their first contributions to TODO:

## Contributing Guidelines

* [OSPO Case Study](https://todogroup.org/guides/casestudies/todo-contribution-guidelines/)
* [TODO Guides](https://todogroup.org/guides/todo-guides-contribution-guidelines/)
* [Whitepapers](https://todogroup.org/guides/whitepaper-guidelines/)
* [OSPOlogy Meetings](https://github.com/todogroup/ospology/tree/main/meetings#what-are-ospology-monthly-meetings)
* [OSPO Survey](https://github.com/todogroup/osposurvey/tree/master/2022#how-to-contribute)